lib/ReadsUtils/ReadsUtilsImpl.py

```diff
@@ -625,6 +598,56 @@ def validateFASTA(self, ctx, params):
         # return the results
         return [out]
 
+    def calculate_fq_stats(self, reads_object, file_path):
+        eautils = kb_ea_utils(self.callback_url)
+        ea_report = eautils.get_ea_utils_stats({'read_library_path':file_path})
+#        print "Full Report : {}".format(ea_report)
+
+        report_lines = ea_report.splitlines()
+        report_to_object_mappings = {'reads':'read_count',
+                                     'total bases':'read_size',
+                                     'len mean':'read_length_mean',
+                                     'len stdev':'read_length_stdev',
+                                     'phred':'phred_type',
+                                     'dups':'number_of_duplicates',
+                                     'qual min':'qual_min',
+                                     'qual max':'qual_max',
+                                     'qual mean':'qual_mean',
+                                     'qual stdev':'qual_stdev'}
+        integer_fields = ['read_count','read_size','number_of_duplicates']
+        for line in report_lines:
+            line_elements = line.split()
```

Comment:
This looks like the eautils wrapper just returns a string with the contents of what is presumably stdout or file output from eautils. eautils should be modified to do the parsing itself and return a sane data structure with that information. There's no reason for the eautils wrapper to force everyone that uses it to do the parsing themselves.